contrib/config/nacos/config.go

```diff
@@ -42,34 +32,6 @@ func WithDataID(dataID string) Option {
 	}
 }
 
-// WithLogDir With nacos config group.
-func WithLogDir(logDir string) Option {
-	return func(o *options) {
-		o.logDir = logDir
-	}
-}
-
-// WithCacheDir With nacos config cache dir.
-func WithCacheDir(cacheDir string) Option {
-	return func(o *options) {
-		o.cacheDir = cacheDir
-	}
-}
-
-// WithLogLevel With nacos config log level.
-func WithLogLevel(logLevel string) Option {
-	return func(o *options) {
-		o.logLevel = logLevel
-	}
-}
-
-// WithTimeout With nacos config timeout.
-func WithTimeout(time time.Duration) Option {
-	return func(o *options) {
-		o.timeoutMs = uint64(time.Milliseconds())
-	}
```

Comment:
This is the option api exposed to the user, bring a bit of your own thinking and don't rely on IDE hints.

Reply:
A better way is to add the relevant Option tests, which will make your linter happy.

Reply:
This option is a parameter of the nacos client, the nacos client is created by the user and passed into newsource function, so this option is unused

Reply:
If you want to use this option, it will need to re-implement the newsource method

Reply:
Sorry, I misunderstood, it turns out that the option defined here is not used at all😅